torch/csrc/jit/mobile/import.cpp

```diff
@@ -171,11 +171,18 @@ std::unordered_set<std::string> BytecodeDeserializer::
   for (const auto& op : ops_list) {
     auto op_item = op.toTuple()->elements();
     TORCH_CHECK(
-        op_item.size() == 2, "There should be two parts in an operator name.");
```

Comment:
Ignore my previous comment about existing runtime being able to load newer models. I didn't realize that we explicitly check for number of elements `== 2`.

Reply:
I updated the code from `>= 2` to `==2 || == 3`. I think I need to change it back to `>=2` for future extension. If the extension (for example, the 4th field) is optional, `>=2` would not throw error, but neglect the extended fields. It can be useful for forward compatibility.